docs/tutorial.md

```diff
@@ -376,6 +376,37 @@ the scheduler starts up, it detects that it is already running 3
 `data` tasks, and starts 2 more in order to reach the goal state of 5.
 We should observe two more tasks starting, `data-3` and `data-4`.
 
+## Task Configuration management
+
+While the above describes configuration of the scheduler itself via
+environment variables, there's also a need for configuration of the
+underlying service tasks themselves in a flexible way.
+
+(Note: As of this writing, the following is still a work in progress,
+but should be available in the next week or two.)
+
+To simplify the common task of getting user-facing configuration to
+service tasks, the developer may follow the following convention in
+naming the environment variables for these configuration options:
+`TASKCFG_<TASK_TYPE>_<NAME>`
+
+For example, an option for tasks of type `index-mgr` could be named
+`TASKCFG_INDEX_MGR_FOO`, while an option for tasks of type `data`
+could be named `TASKCFG_DATA_FOO`. These configuration options will
+automatically be forwarded to the environments of the matching tasks
+as environment variables, with the `TASKCFG_<TASK_TYPE>_` prefixes
+removed. A special prefix of `TASKCFG_ALL_<NAME>` may be used for
+any options that should be passed to *every* task type.
+
+A common need for service developers is an easy way to write
+configuration files before launching tasks. To fulfill this need,
+the developer may provide configuration file template(s) in their
+TaskSet(s). These templates follow the [mustache](https://mustache.github.io/)
+templating format, similar to what's used in DC/OS packaging. The
+templates will be automatically rendered against the task's
+environment (which is customized as described above), and then
```

Comment:
Template contains parameter `{{FOO}}` which is populated with the content of environment variable `FOO`